run_benchmarks.py

```
@@ -1,25 +1,23 @@
-    print("Running STT Benchmark...")
-    os.chdir("stt")
-    subprocess.run([sys.executable, "run_stt.py"])
-    os.chdir("..")
-
-    print("Benchmark suite completed!")
-
+    
+    # Run TTS benchmark first
+    print("\nRunning TTS Benchmark...")
+    
+    subprocess.run([sys.executable, 'run_tts.py'])
```

Ruff (0.15.7)

[error] `subprocess` call: check for execution of untrusted input

(S603)